# Request fields
int8       interaction_id
Trajectory observed_trajectory
---
# Response fields
bool success
